python/sqlflow_submitter/db.py

```diff
@@ -226,10 +247,11 @@ def reader():
                     feature = read_feature(row[field_names.index(name)],
                                            feature_specs[name], name)
                     features.append(feature)
+
                 if label_idx is None:
-                    yield (tuple(features), )
+                    yield list(row), tuple(features), None
```

Comment:
Yield one more duplicated data may not be efficient. Can we pass the full list of selected field to `db_generator`'s `feature_column_names` argument, then form a feature data just before call `predict`?

Reply:
Update as the following logic:
``` python
gen = db.generator()
for row, label in gen():
    features = db.read_features_from_row(row)
    predict(features, label)
```